tox.ini

```diff
@@ -0,0 +1,22 @@
+[tox]
+envlist = ui-tests, flake8
+skipsdist = True
+
+[testenv:ui-tests]
+passenv = DISPLAY
+deps = -rfrontend/test/ui/requirements/requirements.txt
+commands =
+    ./run-integration-tests.sh
+    pytest -v --driver=Firefox --firefox-path=firefox/firefox \
```

Comment:
Instead of using `--firefox-path` can you just ensure the correct version is in the system path? This command line option is deprecated in pytest-selenium and will be removed in a future version.